server/views/pages/dateAndTime.njk

```diff
@@ -195,10 +193,15 @@
       }) }}
     {% endfor %}
     
-        {{ govukButton({
-          text: "Continue"
-        }) }} 
+    {{ govukButton({
```

Comment:
Moved this around so Continue button doesn't show if there are no slots to choose.